README.md

```diff
@@ -67,38 +73,64 @@ plugins:
 custom:
   vpcDiscovery:
     vpcName: '${opt:env}'
-    subnetNames: # optional if securityGroupNames are specified
-      - '${opt:env}_<name of subnet>'
-    securityGroupNames: # optional if subnetNames are specified
-      - '${opt:env}_<name of security group>'
+    # optional if `securityGroups` option is specified
+    # list of tag key and values 
+    subnets:
+      # tag key
+      - tagKey: <tag_name>
+        # an array of values
+        tagValues:
+          - '<tag_value>'
+    # optional if `subnets` option is specified
+    # list of tag key and value or names
+    securityGroups:
+      # tag key
```

Comment:
Nitpick: This comment doesn't add much since it just repeats the field name `tagKey`